content/en/docs/tasks/observability/distributed-tracing/mesh-and-proxy-config/index.md

```diff
@@ -130,13 +130,15 @@ spec:
   template:
     metadata:
       ...
-      proxy.istio.io/config: |
-        tracing:
-          sampling: 10
-          custom_tags:
-            my_tag_header:
-              header:
-                name: host
+      annotations:
+       ...
+       proxy.istio.io/config: |
+         tracing:
+           sampling: 10
+           custom_tags:
+             my_tag_header:
+               header:
+                 name: host
```

Comment:
I believe the indentation needs to be 2 more spaces?

Reply:
Thanks for the PR and we also need the CLA.

Reply:
> I believe the indentation needs to be 2 more spaces?

fixed